install/templateservicebroker/apiserver-template.yaml

```diff
@@ -40,7 +40,7 @@ objects:
           image: ${IMAGE}
           imagePullPolicy: IfNotPresent
           command:
-          - "/usr/bin/openshift"
+          - "/usr/bin/template-service-broker"
```

Comment:
@deads2k 
```
Error: unknown flag: --loglevel
Usage:
  template-service-broker [flags]
```

seems like something else needs to be done in the new template-service-broker binary to add loglevel flags?